tools/api_boost/testdata/decl_ref_expr.cc.gold

```diff
@@ -17,5 +20,8 @@ public:
     default:
       break;
     }
+    API_NO_BOOST(envoy::config::overload::v2alpha::Trigger) foo;
```

Comment:
In this case api_booster should keep `#include "envoy/config/overload/v2alpha/overload.pb.h"` as well?

Reply:
Good point, fixed.